packages/tool-cache/src/tool-cache.ts

```diff
@@ -662,7 +662,7 @@ function _isExplicitVersion(versionSpec: string): boolean {
   return valid
 }
 
```

Comment:
We should doc these functions as well, like we do our other public functions, ie:

```
/**
 * Finds the path to a tool version in the local installed tool cache
 *
 * @param toolName      name of the tool
 * @param versionSpec   version of the tool
 * @param arch          optional arch.  defaults to arch of computer
 */
```